src/collective/fingerpointing/logger.py

```diff
@@ -23,6 +23,7 @@
     logfile = AUDITLOG
 
 logger = logging.getLogger(PROJECTNAME)
+logger.setLevel(logging.INFO)
```

Comment:
that's odd. please check out the branch, and modify buildout.cfg by adding

```ini
[instance]
event-log-level = ERROR
```

now running bin/instance and modifying a content object will lead to
* no audit log messages in `var/log/instance.log`
* but audit log messages in `var/log/audit.log`

for me, this works.